Makefile

```diff
@@ -247,7 +247,8 @@ help:
 	@echo "make sso/setup                   local keycloak instance"
 	@echo "make sso/config                  local configure realm"
 	@echo "make sso/teardown                teardown keycloak instance"
-	@echo "make redhatsso/setup             setup mas sso clientId & clientSecret "
+	@echo "make redhatsso/setup             setup mas sso clientId & clientSecret"
+	@echo "make imagepull/secret/setup      setup dockerconfig image pull secret"
```

Comment:
I'd like the names around this to make clear that it is to setup the image pull secret for the dataplane side